zomes/rea_economic_resource/lib/src/lib.rs

```diff
@@ -95,6 +95,17 @@ impl API for EconomicResourceZomePermissableDefault {
             hdk::prelude::debug!("create_inventory_from_event::contained_in index {:?}", e);
         };
 
+        // TODO: create index depending on the event params
+        // check if produce action
+
+        let produce_action = get_builtin_action("produce").unwrap();
+        let raise_action = get_builtin_action("raise").unwrap();
+        let event_action = String::from(event_params.action.to_owned());
+        if event_action == produce_action.id || event_action == raise_action.id {
```

Comment:
instead of checking like this here, just check if the new economic resource has a set primary_accountable field (Some not None). That way we consolidate the actions check